src/Controls/src/Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -193,25 +237,42 @@ static void SetModalPresentationStyle(BindableObject element, UIModalPresentatio
 		public static readonly BindableProperty PrefersHomeIndicatorAutoHiddenProperty =
 			BindableProperty.Create(nameof(PrefersHomeIndicatorAutoHidden), typeof(bool), typeof(Page), false);
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='GetPrefersHomeIndicatorAutoHidden']/Docs/*" />
+		/// <summary>
+		/// Gets a Boolean that indicates whether is allowed to hide the visual indicator for returning to the Home Screen.
+		/// </summary>
+		/// <param name="element">A page, the VisualElement that occupies the entire screen.</param>
+		/// <returns></returns>
 		public static bool GetPrefersHomeIndicatorAutoHidden(BindableObject element)
 		{
 			return (bool)element.GetValue(PrefersHomeIndicatorAutoHiddenProperty);
 		}
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='SetPrefersHomeIndicatorAutoHidden'][1]/Docs/*" />
+		/// <summary>
+		/// Sets a Boolean that indicates whether is allowed to hide the visual indicator for returning to the Home Screen.
+		/// </summary>
+		/// <param name="element">A page, the VisualElement that occupies the entire screen.</param>
+		/// <param name="value">True when hide the home indicator, or false if you want the indicator to show at all times.</param>
```

Comment:
For the words "true" and "false", you can improve the documents by adding <see langword="true"/>.

We should also avoid using "you" in the sentence.